benchmarks/dynamo/huggingface.py

```diff
@@ -98,6 +98,8 @@ def pip_install(package):
     # Fails with even batch size = 1
     "GPTJForCausalLM",
     "GPTJForQuestionAnswering",
+    # Out of memory in CI
+    "OPTForCausalLM",
```

Comment:
Better to add to this list if this is a CI-only failure, https://github.com/pytorch/pytorch/blob/333e771394998e6eb1d8b451cae22717123c07fc/benchmarks/dynamo/common.py#L148

Reply:
Changed